src/sass/foundation/variables.scss

```diff
@@ -122,6 +122,43 @@ $buttonTerBorderColor:     $green3 !default;
 $buttonTerTextColor:       $buttonColor !default;
 $buttonTerDisabledTextColor:   darken($green3, 20%) !default;
 
+//Alternatives buttons
+$altButtonDefaultBgColor : $transparent !default;
+$altButtonDefaultBorder: $grayLightLight !default;
+$altButtonDefaultTextColor: $grayLightLight !default;
+$altButtonDefaultHoverColor : $grayLightLight !default;
+$altButtonDefaultHoverTextColor : $white !default;
+$altButtonDefaultDisabledTextColor : lighten($grayLightLight, 10%) !default;
+
+$altButtonPrimaryBgColor : $transparent !default;
+$altButtonPrimaryBorder: $blue !default;
```

Comment:
The value of `$altButtonPrimaryBorder ` should be `$buttonPrimaryBorderColor ` 
ect...